packages/ols/ols.yaml

```
@@ -0,0 +1,33 @@
+---
+name: ols
+description: Language server for Odin. This project is still in early development.
+homepage: https://github.com/DanielGavin/ols
+licenses:
+  - MIT
+languages:
+  - Odin
+categories:
+  - LSP
+
+source:
+  id: pkg:github/DanielGavin/ols@nightly
+  asset:
+    - target: darwin_x64
```

This binary is executable on Apple chips as well via [Rosetta](https://en.wikipedia.org/wiki/Rosetta_(software))

```suggestion
    - target: [darwin_x64, darwin_arm64]
```